client/src/app/pages/applications/manage-imports-details/manage-imports-details.tsx

```diff
@@ -74,6 +74,7 @@ export const ManageImportsDetails: React.FC = () => {
   const tableControls = useLocalTableControls({
     tableName: "manage-imports-details",
     idProperty: "Application Name",
+    dataNameProperty: "Application Name",
```

Comment:
I am probably missing something...
Should it be one of the `columnNames` value?

Reply:
Also, while the `columnNames` for the table properties often line up with the actual item field name, there is no requirement that it do so.  In some tables, the columns are "UI synthetic" being derived from multiple data sources.

That's just a long way of saying the `idProperty` and `dataNameProperty` values do not need to have a corresponding entry in `columnNames`.